src/Microsoft.Sbom.Targets/Microsoft.Sbom.Targets.props

```diff
@@ -1,27 +1,21 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-
-  <PropertyGroup>
-    <GenerateSbomTask_TFM Condition=" '$(TargetFramework)' == 'net6.0' ">net6.0</GenerateSbomTask_TFM>
-      <GenerateSbomTask_TFM Condition=" '$(TargetFramework)' == 'net8.0' ">net8.0</GenerateSbomTask_TFM>
-  </PropertyGroup>
-  <UsingTask TaskName="Microsoft.Sbom.Targets.GenerateSbomTask" AssemblyFile="$(MSBuildThisFileDirectory)\..\tasks\$(GenerateSbomTask_TFM)\Microsoft.Sbom.Targets.dll" />
-
   <PropertyGroup>
+        <GenerateSBOM Condition=" '$(GenerateSBOM)' == '' ">false</GenerateSBOM>
         <!-- TODO in practice these will default to existing nuget props instead of these test values.
             Need to consult with .NET SDK/ PM team for which existing props will make sense as defaults. -->
-        <SbomGenerationBuildDropPath>C:\Users\gustavoca\Repos\github\sbom-test\src\GenerateSBOMTest\GenerateSBOMTest\bin\Debug\net8.0</SbomGenerationBuildDropPath>
-        <SbomGenerationBuildComponentPath>$(MSBuildProjectDirectory)</SbomGenerationBuildComponentPath>
-        <SbomGenerationPackageSupplier>test-supplier</SbomGenerationPackageSupplier>
-        <SbomGenerationPackageName>test-name</SbomGenerationPackageName>
-        <SbomGenerationPackageVersion>1.0.0</SbomGenerationPackageVersion>
-        <SbomGenerationNamespaceBaseUri>http://spdx.org/spdxdocs/$(SbomGenerationPackageName)</SbomGenerationNamespaceBaseUri>
-        <SbomGenerationNamespaceUriUniquePart></SbomGenerationNamespaceUriUniquePart>
-        <SbomGenerationExternalDocumentReferenceListFile></SbomGenerationExternalDocumentReferenceListFile>
-        <SbomGenerationFetchLicenseInformation>false</SbomGenerationFetchLicenseInformation>
-        <SbomGenerationEnablePackageMetadataParsing>false</SbomGenerationEnablePackageMetadataParsing>
-        <SbomGenerationVerbosity>LogAlways</SbomGenerationVerbosity>
+        <SbomGenerationBuildDropPath Condition=" '$(SbomGenerationBuildDropPath)' == '' ">$(OutDir)</SbomGenerationBuildDropPath>
```

Comment:
nit: I think these conditionals might not be needed as we are initializing these properties with their default values. The user can override these values by directly stating `<SbomGenerationBuildDropPath> custom path </SbomGenerationBuildDropPath>` in their .csproj after importing this props file.

Reply:
Jk ignore me 😊